Components/Games/Game.js

```diff
@@ -47,6 +47,8 @@ function Game(props) {
 
     let formattedTime = moment.utc(timeDifference).format('HH:mm');
 
+    const title = game.event.name ? `${game.name} - ${game.event.name}` : game.name;
```

Comment:
I suggest we put the event´s name in front of the game´s name as a prefix to make it more visible when looking at the list of games in the lobby. What do you think?